### PR TITLE
mch stores

### PR DESCRIPTION
**Pull request by Injiri**
1. Proper UI resource imports
2. import fix for breaking autocomplete select

### CODE REVIEW

**Comment by Injiri** [comment]
can close  on this, I see some changes already in upstream

**Comment by ningosi** [comment]
@Injiri would you check on the conflicts please

File: omod/src/main/webapp/pages/stores.gsp [hunk 1]
```diff
@@ -5,11 +5,11 @@
     ui.includeCss("uicommons", "datatables/dataTables_jui.css")
     ui.includeJavascript("ehrconfigs", "emr.js")
     ui.includeCss("ehrconfigs", "referenceapplication.css")
+    ui.includeCss("registration", "onepcssgrid.css")
```
Comment (ningosi) [inline omod/src/main/webapp/pages/stores.gsp]: do we have such provider here? can we get that from ehrconfigs

Comment (ningosi) [reply omod/src/main/webapp/pages/stores.gsp]: get **onepcssgrid**.css from ehrconfigs

**Review by ningosi — COMMENTED** [review]
try to get as much resources from ehrconfigs unless it is not there

**Review by ningosi — APPROVED** [review]

**Pull request merged**